website/content/api-docs/auth/saml.mdx

```diff
@@ -51,6 +51,15 @@ Configures the auth method with a SAML identity provider.
   according to the current logging level. When `verbose_logging` is `true`,
   debug-level logs provide user attributes and trace-level logs provide the full
   SAML response.
+- `validate_response_and_assertion_signatures` `(bool, false)` - Enables validation
+  of signatures for both response and assertion in the SAML response. If IDP allows
+  signing both response and assertion then **recommendation** is to opt for this option.
```

Comment:
Is the recommendation is to `opt in`?

```suggestion
  signing both response and assertion then **recommendation** is to opt in for this option.
```

Reply:
removed the `validate_response_and_assertion_signatures` option